### DISCUSSION TITLE
Does SMPlayer support TTML subtitle files?

### DISCUSSION THREAD

**Discussion by umwnp** [smplayer-dev/smplayer#1013] [Show and tell]
There are translation files on YouTube and other platforms, and I think that SMPlayer does not support this type of file.
Only: 
Subtitles (*.srt *.sub *.ssa *.ass *.idx *.txt *.smi *.rt *.utf *.aqt *.vtt *.mpsub)

**Comment by smplayer-dev** [comment]
Unfortunately, it seems that neither mpv nor mplayer supports this format. It is up to their developers to add support for it.
